src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,70 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [toggleState, setToggle] = useState(tabs[0]);
+
+  const chooseTab = (tab: Tab) => {
+    setToggle(tab);
+  };
+
+  const selectedTab: Tab = tabs.find(
+    tab => toggleState.id === tab.id,
+  ) || tabs[0];
+
+  const tabClick = (tab: Tab) => {
+    if (toggleState.id !== tab.id) {
+      chooseTab(tab);
+    }
+  };
+
+  return (
+
+    <>
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {toggleState.title}
+      </h1>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map((tab) => (
+              <li
+                key={tab.id}
+                className={toggleState.id === tab.id ? 'is-active' : ''}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={(e) => {
+                    tabClick(tab);
+                    e.preventDefault();
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/86923089/518f504d-a45a-47b3-b9f5-3346dc5f0279)

because of this selected tab doesn't represented in the url